nginx-controller/controller/controller.go

```diff
@@ -1271,6 +1297,17 @@ func (lbc *LoadBalancerController) createMergableIngresses(master *extensions.In
 		return &mergeableIngresses, err
 	}
 
+	for key, val := range master.Annotations {
+		if key == "nginx.org/mergible-ingress-type" || !strings.Contains(key, "nginx.org/") {
+			continue
+		}
+		if strings.Contains(val, "serviceName=") {
```

Comment:
Master configuration should be able to take any annotation that is not only for a specific location.